clients/pkg/promtail/client/batch_test.go

```diff
@@ -164,3 +164,23 @@ func TestHashCollisions(t *testing.T) {
 		assert.Equal(t, ls1.String(), req.Streams[1].Labels)
 	}
 }
+
+// store the result to a package level variable
+// so the compiler cannot eliminate the Benchmark itself.
+var result string
+
+func BenchmarkLabelsMapToString(b *testing.B) {
+	labelSet := make(model.LabelSet)
+	labelSet["label"] = "value"
+	labelSet["label1"] = "value"
+	labelSet["label2"] = "value"
+	labelSet["__tenant_id__"] = "value"
```

Comment:
```suggestion
	labelSet["__tenant_id__"] = "another_value"
```

Changing this to avoid tripping `goconst` linter suggesting we make `"value"` a constant